### ISSUE TITLE
Dahed Line rule in Boundary Point is unclear

### ISSUE THREAD

**Issue by cniemeyerBMW** [OpenSimulationInterface/open-simulation-interface#205]
The following rule in the BoundaryPoint Documentation should be removed:

"For dashed lines, one BoundaryPoint has to be at the start and another at the end of each dashed line segment."

Reason: From this definition it is unclear where the white and the "black" part of a dashed line is. Moreover it will result into unnecessary amounts of data, when sending laneboundaries.

Suggestion: Remove the rule and create new attributes in LaneBoundary (e.g. dashed lane length, spacing, initial offset, ...). If a dashed line pattern changes, create new LaneBoundary.

**Comment by HendrikAmelunxen** [comment]
I agree to this suggestion. Such a representation for dashed lines would be clear and good to handle.

**Comment by carsten-kuebler** [comment]
Update of documentation is possible for v3.0.1. A change of the dashed line semantic would lead to V4.0.0. We could add a new enum TYPE_DASHED_LINE_HOMOGENOUS and additional fields for V3.1.

The same would apply to TYPE_BOTTS_DOTS.

**Comment by carsten-kuebler** [comment]
The PR improves the documentation for V 3.0.1

**Comment by pmai** [comment]
Moved to 4.0.0, since major changes not possible for 3.2.0 release.
